src/actions/suggestions.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+export default function suggestionsReducer (state, action) {
```

Comment:
There is no equivalent of `combineReducers` in this model.  I have thought about how I would implement one, but TBH I think it doesn't have a great place in here.  In fact the intention is to keep all this logic in the container component because it is simple and clear where everything comes from.